src/compose/service.coffee

```diff
@@ -14,7 +14,7 @@ validRestartPolicies = [ 'no', 'always', 'on-failure', 'unless-stopped' ]
 PORTS_REGEX = /^(?:(?:([a-fA-F\d.:]+):)?([\d]*)(?:-([\d]+))?:)?([\d]+)(?:-([\d]+))?(?:\/(udp|tcp))?$/
 
 parseMemoryNumber = (numAsString, defaultVal) ->
-	m = numAsString?.toString().match(/^([0-9]+)([bkmg]?)$/)
+	m = numAsString?.toString().match(/^([0-9]+)([bkmg])?b?$/i)
```

Comment:
NIT: This regex allows `bb` but I don't think it actually matters! The function should still behave as if it is given a single `b`